management/server/http/api/openapi.yml

```diff
@@ -929,6 +933,7 @@ components:
         - activity
         - activity_code
         - initiator_id
+        - initiator_email
```

Comment:
I guess we also keep track of target_email. Do we need to return it in the response?

Reply:
Not exactly. We response the target email in the "meta" map because different type of actions point to different type of resource. For example if I delete a DNS setting or a route setting then make no sense to send target_email is we do not talk about a user.

Reply:
Thank you for the clarification. I think it is fine then.